CMakeLists.txt

```
@@ -297,8 +297,8 @@ set_option(SDL_DISKAUDIO           "Support the disk writer audio driver" ON)
 set_option(SDL_DUMMYAUDIO          "Support the dummy audio driver" ON)
 set_option(SDL_DUMMYVIDEO          "Use dummy video driver" ON)
 dep_option(SDL_IBUS                "Enable IBus support" ON ${UNIX_SYS} OFF)
-set_option(SDL_OPENGL              "Include OpenGL support" ON)
-set_option(SDL_OPENGLES            "Include OpenGL ES support" ON)
+set_option(SDL_OPENGL              "Include OpenGL support" ON "VISIONOS" OFF)
```

I think `"VISIONOS"` is evaluating to true, since it's a string. This is what's causing the build errors - OpenGL is disabled for all platforms. Maybe try just `VISIONOS`?

---

Assuming enable opengl by default for all non-visionos, and force disable for vision (with no way to re-enable it)

```suggestion
dep_option(SDL_OPENGL              "Include OpenGL support" ON "NOT VISIONOS" OFF)
```